test/suite/test_schema06.py

```diff
@@ -42,6 +42,10 @@ class test_schema06(wttest.WiredTigerTestCase):
         ('lsm', { 'idx_config' : ',type=lsm' }),
     ])
 
+    # Enable statistics collection for session level schema lock statistics test.
```

Comment:
comment can be updated, currently we don't have enable/disable for session level statistics.